src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -3,6 +3,22 @@ title: Using packages
 description: How to use packages in your Flutter app.
 ---
 
+**Plugin**
+A plugin is ready to use software component that can be added to existing 
+software for more features. It is about making native functionality 
+available to Flutter.
+
+**Package**
+A package is about integrating Flutter with an existing native application.
```

Comment:
I think this is misleading.  How about:
```
***Packages***
A package contains shared and reusable code and includes a list of
dependencies (specified in the `pubspec.yml` file), and (optionally)
Dart libraries, apps, resources, tests, images, and examples.
The pub.dev site lists many packages developed by Google engineers
and other members of the Flutter and Dart community that you can use in your app.
```

Also, I'd place the definition for packages BEFORE plugins, as plugins are a TYPE of package.

```
***Plugins***
A plugin package is a special kind of package that makes platform functionality
available to the app. Plugin packages can be written for Android (using Kotlin or Java),
iOS (using Swift or Objective-C), web, macOS, Windows, or Linux,
or any combination thereof. An example of a plugin would provide the Flutter app
the ability to access a device's camera.
```